### PR TITLE
W-15261565 bot 1.6.2 rns

### PR DESCRIPTION
**Pull request by tasmoore**

### CODE REVIEW

**Review by justus-schach — DISMISSED** [review]
LGTM

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,60 @@
+= RPA Bot 1.6.2 Release Notes
+
+RPA Bot is a software robot that runs in an on-premise environment and executes the assigned automations.
+
+include::partial$release-notes/rn-last-version-message.adoc[] 
+
+== March , 2024
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc]:
```suggestion
== March 25, 2024
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+== March 25, 2024
+
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+1]
+
+[%header,cols="2a,1a"]
+|===
+| Issue Resolution | ID
+|The custom message created from the *Write Bot Message* action step and now displays the full 2,000 characters and no longer truncates the message after 1,000 characters.| W-15252227
```
Comment (Crispy-Salesforce) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc]:
```suggestion
|The custom message is created from the *Write Bot Message* action step. Now it displays the full 2,000 characters and no longer truncates the message after 1,000 characters.| W-15252227 
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+|RPA Bot now removes sessions that are in a state of "Disconnected" to prevent RPA Bot from attempting to connect to and create Secure Sessions from "Disconnected" sessions.| W-15241413
+|===
+
+== February 21, 2024
+
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]
+
+* The System Settings panel now has a `Bot Service State` entry, which shows the current Windows Service state and configuration.
+* The *Password* and *Current User* fields on the Session Settings panel are renamed *Default Session User Password* and *RPA Bot Configurator User* respectively.
```
Comment (Crispy-Salesforce) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc]:
```suggestion
* The *Password* and *Current User* fields on the Session Settings panel are renamed to *Default Session User Password* and *RPA Bot Configurator User* respectively.
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+* A warning message has been added to indicate that if a deployed process is running on the bot instance, the process may be canceled if RPA Bot is restarted as a result of a change made to the RPA Bot Configurator.
```
Comment (Crispy-Salesforce) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc]:
```suggestion
* A warning message is added to indicate that if a deployed process is running on the bot instance, the process can be canceled if RPA Bot is restarted as a result of a change made to the RPA Bot Configurator.
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+* When a change is made to the RPA Bot Configurator while a process is running, you can now select from multiple restart, exit, and cancel options to decide how you want to proceed with the process that is running and the changes to the RPA Bot Configurator.
```
Comment (Crispy-Salesforce) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc]:
```suggestion
* When a change is made to the RPA Bot Configurator while a process is running, you can now select from multiple restart, exit, and cancel options to decide how to continue with the process that is running and the changes to the RPA Bot Configurator.
```

Comment (Crispy-Salesforce) [reply rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc]: Is the use of "the" consistent? Docs names "RPA Bot" without "the", which seems to be OK as this is a proper name. But docs use "the" with "RPA Bot Configurator", which seems to be a proper name as well as it is capitalized.

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+* The RPA Bot Configurator now includes a Diagnostic Infos panel to help you identify current and potential problems with RPA Bot.
+* Redundant third party assemblies have been removed.
```
Comment (Crispy-Salesforce) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc]:
```suggestion
* Redundant third-party assemblies are removed.
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+* Dynamic process execution throttling has been added to reduce the number of times RPA Bot attempts to restart a failing process.
```
Comment (Crispy-Salesforce) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc]:
```suggestion
* Dynamic process execution throttling is added to reduce the number of times RPA Bot attempts to restart a failing process.
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+* You can now view the path to the analysis packages on the default run result message in RPA Manager if the setting to write an analysis packages is enabled on RPA Bot.
```
Comment (Crispy-Salesforce) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc]:
```suggestion
* You can now view the path to the analysis packages on the default run result message in RPA Manager if the setting to write an analysis package is enabled on RPA Bot.
```

**Review by Crispy-Salesforce — DISMISSED** [review]

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc (hunk 2 above)
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc]:
```suggestion
|The custom message created from the *Write Bot Message* action step now displays the full 2,000 characters and no longer truncates the message after 1,000 characters.| W-15252227 
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+|===
+
+== February 21, 2024
+
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]
+
+* The System Settings panel now has a `Bot Service State` entry, which shows the current Windows Service state and configuration.
+* The *Password* and *Current User* fields on the Session Settings panel are renamed to *Default Session User Password* and *RPA Bot Configurator User* respectively.
+* A warning message has been added to indicate that if a deployed process is running on the bot instance, the process may be canceled if RPA Bot is restarted as a result of a change made to the RPA Bot Configurator.
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc]:
```suggestion
* A warning message has been added to indicate that if a deployed process is running on the bot instance, the process might cancel if RPA Bot is restarted as a result of a change made to the RPA Bot Configurator.
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+|===
+
+== February 21, 2024
+
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]
+
+* The System Settings panel now has a `Bot Service State` entry, which shows the current Windows Service state and configuration.
+* The *Password* and *Current User* fields on the Session Settings panel are renamed to *Default Session User Password* and *RPA Bot Configurator User* respectively.
+* A warning message has been added to indicate that if a deployed process is running on the bot instance, the process might cancel if RPA Bot is restarted as a result of a change made to the RPA Bot Configurator.
```
Comment (Crispy-Salesforce) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.6.2-release-notes.adoc]:
```suggestion
* A warning message is added to indicate that if a deployed process is running on the bot instance, the process might cancel if RPA Bot is restarted as a result of a change made to the RPA Bot Configurator.
```

**Review by Crispy-Salesforce — DISMISSED** [review]

**Review by Crispy-Salesforce — APPROVED** [review]

**Pull request merged**